prow/config/tide_test.go

```diff
@@ -148,6 +148,80 @@ func TestMergeMethod(t *testing.T) {
 		}
 	}
 }
+func TestMergeTemplate(t *testing.T) {
+	ti := &Tide{
+		MergeTemplate: map[string]TideMergeCommitTemplate{
+			"kubernetes/kops": TideMergeCommitTemplate{
+				TitleTemplate: "",
+				BodyTemplate:  "",
+			},
+			"kubernetes/charts": TideMergeCommitTemplate{
+				TitleTemplate: "{{ .Number }}",
+				BodyTemplate:  "",
+			},
+			"helm/charts": TideMergeCommitTemplate{
+				TitleTemplate: "",
+				BodyTemplate:  "{{ .Body }}",
+			},
+			"kubernetes-helm": TideMergeCommitTemplate{
+				TitleTemplate: "{{ .Title }}",
+				BodyTemplate:  "{{ .Body }}",
+			},
+		},
+	}
+
+	var testcases = []struct {
+		org      string
+		repo     string
+		expected TideMergeCommitTemplate
+	}{
+		{
+			"kubernetes",
+			"kubernetes",
+			TideMergeCommitTemplate{},
+		},
```

Comment:
nit: Please specify the field names for these struct literals.

Reply:
Does this mean
```
TideMergeCommitTemplate{}
```
is equivalent to
```
TideMergeCommitTemplate{
  Title: nil,
  Body: nil,
}
```
? I guess fields in the struct are set to their default values?